### PR TITLE
Tim

### PR DESCRIPTION
**Pull request by TimothyMather**
Merge ble and radio https://trello.com/c/IHBC9YGj

### CODE REVIEW

**Comment by s-axford** [comment]
Since we hope to have the code become usable on a variety of systems via open source perhaps "combined" in not the best naming scheme for the compilation folder. We could have a folder called builds/ then inside that have a folder called rfm69-nrf52/. then later system builds can be added by naming them rfm23-m0 for example (RFM23 board with m0 microcontroller).

Also this PR results in alot of duplicated code being in the radio or bluetooth folders but also the combined folders. not sure how we want to deal with that

**Comment by TimothyMather** [comment]
Updated the name, I'm planning on removing the duplicate code once the simulator is up to date with the combined code

**Review by s-axford — APPROVED** [review]

**Comment by JeremyButt** [comment]
[Phantom messages](https://trello.com/c/9kbGqVGT/33-phantom-messages)

**Comment by JeremyButt** [comment]
[Integrate BLE and Radio](https://trello.com/c/IHBC9YGj/30-integrate-ble-and-radio)

**Comment by JeremyButt** [comment]
[Integrate BLE and Radio](https://trello.com/c/IHBC9YGj/30-integrate-ble-and-radio)

File: bluetooth/nrf52/nrf52.ino [hunk 1]
```diff
@@ -1,23 +1,36 @@
 
 #include "NRF52RxTx.h"
```
Comment (JeremyButt) [inline bluetooth/nrf52/nrf52.ino]: it looks like this file is no longer used as it is copied to nrf52-rfm69/nrf52-rfm69.ino. @DelightedWaif please remove

**Comment by JeremyButt** [comment]
@DelightedWaif looks good other than the old files that were copied needing to be removed

**Comment by TimothyMather** [comment]
@JeremyButt I removed the unused files

File: nrf52-rfm69/ChirpFMRxTx.h [hunk 2]
```diff
@@ -2,6 +2,6 @@ class ChirpFMRxTx
 {
```
Comment (JeremyButt) [inline nrf52-rfm69/ChirpFMRxTx.h:2]: @s-axford @DelightedWaif @johnnyb8 this is def a preference but thoughts on having the interface at this level? maybe up one? or only move it when we have more hardware implementations?

**Review by JeremyButt — APPROVED** [review]
Looks good! :) 💯 🚀 👍 😃

**Pull request merged**